rocketpy/simulation/flight.py

```diff
@@ -3476,6 +3535,20 @@ def add_parachutes(self, parachutes, t_init, t_end):
                 ]
                 self.list += parachute_node_list
 
+        def add_controllers(self, controllers, t_init, t_end):
```

Comment:
The `add_parachutes` and `add_controllers` methods look the same, matbe we should have a single function to do the loop that is being done inside the two functions.

What do you think?

Reply:
They are not the same, there is a small difference when initializing the TimeNode object